test/stat_tracker_test.rb

```diff
@@ -66,7 +66,35 @@ def test_it_can_return_the_worst_coach_of_the_season
     assert_equal "Alain Vigneault", @stat_tracker.worst_coach(20132014)
   end
 
+  def test_it_can_return_lowest_score
+    assert_equal 2, @stat_tracker.lowest_total_score
+  end
+
+  def test_it_can_return_average_goals_per_game
+    assert_equal 4.13, @stat_tracker.average_goals_per_game
+  end
 
+  def test_it_can_return_average_goals_by_season
+    expected = {
+      "20122013"=>3.8,
+      "20162017"=>4.57,
+      "20142015"=>4.5,
+      "20152016"=>3.8,
+      "20132014"=>4.4,
+      "20172018"=>3.67
+      }
+
+    assert_equal expected, @stat_tracker.average_goals_by_season
+  end
+
+  def test_can_return_percentage_of_visitor_wins
+    assert_equal 0.38, @stat_tracker.percentage_visitor_wins
+  end
+
+  def test_can_return_percentage_of_home_wins
+    assert_equal 0.53, @stat_tracker.percentage_home_wins
+  end
+  
   ######  Move these tests somewhere else
```

Comment:
The comment tags are very helpful for my understanding of things during dev process.